src/index.html

```diff
@@ -5,8 +5,320 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="logo"
+      >
+        <img 
+          src="images\logo.png" 
+          alt="logo"
+        >
+      </a>
+ 
+      <nav class="nav">
+        <a class="nav__item is-active" href="#">Apple</a>
+        <a class="nav__item" href="#">Samsung</a>
+        <a class="nav__item" href="#">Smartphones</a>
+        <a 
+          class="nav__item" 
+          href="#"
+          data-qa="nav-hover"
+        >
+          Laptops & Computers
+        </a>
+        <a class="nav__item" href="#">Gadgets</a>
+        <a class="nav__item" href="#">Tablets</a>
+        <a class="nav__item" href="#">Photo</a>
+        <a class="nav__item" href="#">Video</a>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div data-qa="card" class="card catalog__card">
+          <img 
+            src="./images/imac.jpeg" 
+            alt="iMac" 
+            class="card__image"
+          >
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__reviews">
+            <div class="stars stars--5">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+    
+            <p class="reviews">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-sum">$2,199</p>
+          </div>
+          <div class="card__button">
```

Comment:
```suggestion
          </div>
          
          <div class="card__price">
            <p class="card__price-text">Price:</p>
            <p class="card__price-sum">$2,199</p>
          </div>
          
          <div class="card__button">
```
add some blank lines to increase readability